test/mongocontroller/controller.go

```diff
@@ -121,3 +127,14 @@ func (c *Controller) Destroy(deleteTempDir bool) error {
 	}
 	return nil
 }
+
+func getMongoDBVer(ExecutablePath string) (*semver.Version, error) {
```

Comment:
```suggestion
func getMongoDBVer(executablePath string) (*semver.Version, error) {
```

Regular old variables should be lowercase

Reply:
👍

Reply:
:+1: